RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Firefox.java

```diff
@@ -652,6 +668,20 @@ private void getDownloadVersion24() {
                 if (bbart != null) {
                     bbartifacts.add(bbart);
                 }
+                
+                // find the downloaded file and create a TSK_DOWNLOAD_SOURCE for it.
+                 try {
+                    for (AbstractFile downloadedFile : fileManager.findFiles(dataSource, FilenameUtils.getName(downloadedFilePath), FilenameUtils.getPath(downloadedFilePath))) {
+                        BlackboardArtifact downloadSourceArt =  downloadedFile.newArtifact(BlackboardArtifact.ARTIFACT_TYPE.TSK_DOWNLOAD_SOURCE);
+                        downloadSourceArt.addAttribute(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_URL,
+                            NbBundle.getMessage(this.getClass(), "Firefox.parentModuleName"), url)); //NON-NLS
```

Comment:
Change Firefox.parentModule Name to RecentActivityExtracterModuleFactory.getModuleName()